sagemaker-debugger/debugger_interactive_analysis_profiling/interactive_analysis_profiling_data.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Is this necessary? We already have an installation cell for smdebug earlier in this notebook.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>If they skip the initial installation, still we need another cell like this because new NI kernels do not have smdebug by default. If they enable the first installation cells, then this will anyways import smdebug and skip the installation.</p>

Reply:
<p>Understood, but with the new changes above guaranteeing that smdebug is always installed, are these lines still necessary?</p>